Source/WebKit/UIProcess/API/Cocoa/WKUserScript.mm

```diff
@@ -45,7 +45,7 @@ - (instancetype)initWithSource:(NSString *)source injectionTime:(WKUserScriptInj
     if (!(self = [super init]))
         return nil;
 
-    API::Object::constructInWrapper<API::UserScript>(self, WebCore::UserScript { source, { }, { }, { }, API::toWebCoreUserScriptInjectionTime(injectionTime), forMainFrameOnly ? WebCore::UserContentInjectedFrames::InjectInTopFrameOnly : WebCore::UserContentInjectedFrames::InjectInAllFrames, WebCore::WaitForNotificationBeforeInjecting::No }, *contentWorld->_contentWorld);
+    API::Object::constructInWrapper<API::UserScript>(self, WebCore::UserScript { source, { }, { }, { }, API::toWebCoreUserScriptInjectionTime(injectionTime), forMainFrameOnly ? WebCore::UserContentInjectedFrames::InjectInTopFrameOnly : WebCore::UserContentInjectedFrames::InjectInAllFrames}, *contentWorld->_contentWorld);
```

Comment:
Nit: I think the style bot was complaining about a missing space in `WebCore::UserContentInjectedFrames::InjectInAllFrames}`

Reply:
Yeah, forgot to push that. I have it fixed.